src/bin/stashd.rs

```diff
@@ -13,20 +13,17 @@
 
 use clap::derive::Clap;
 use log::*;
-use std::env;
-
-use lnpbp::TryService;
+// use std::env;
 
 use rgb::error::BootstrapError;
-use rgb::stash::{Config, Opts, Runtime};
+use rgb::stash::{Opts, main_with_opts};
 
 #[tokio::main]
 async fn main() -> Result<(), BootstrapError> {
     // TODO: Parse config file as well
     let opts: Opts = Opts::parse();
-    let config: Config = opts.into();
 
-    if env::var("RUST_LOG").is_err() {
+    /* if env::var("RUST_LOG").is_err() {
```

Comment:
why we have to exclude this part?

Reply:
this was because i moved the opts -> config conversion inside `main_with_opts`, so there was no way at that point to set a config field.

now it's been restored like it was before